lib/modules/showImages.js

```diff
@@ -1058,7 +1070,8 @@ function generateImage(options) {
 	options.openInNewWindow = module.options.openInNewWindow.value;
 
 	const element = $(imageTemplate(options))[0];
-	const image = element.querySelector('img');
+	const image = element.querySelector('img.res-image-media');
+	const anchor = element.querySelector('a.res-expando-link');
```

Comment:
For some reason isn't this change included in any of the individual commit diffs.